### ISSUE TITLE
Deprecate/archive and recommend peggy.js as official successor project?

### ISSUE THREAD

**Issue by lionel-rowe** [pegjs/pegjs#675]
As this project is basically dead (last commit 4 years ago) and [peggy.js](https://github.com/peggyjs/peggy/) is the de-facto successor (3 commits in the last few days), it would make sense to make it official, deprecate/archive this repo, and add a link in the README pointing to [peggyjs/peggy](https://github.com/peggyjs/peggy/). Any reason not to do that, other than the maintainer not being active on GitHub any more?

**Comment by hildjj** [comment]
As far as we can tell, there is nobody who has the keys to this repo or to http://pegjs.org that is willing to do anything, or that even still responds to messages.  Several people have tried multiple times.  I don't think there's much more we can do at this point, other than respond to folks who ask questions here and direct them to Peggy.

**Comment by lionel-rowe** [comment]
Fair enough, I guess anyone who starts using pegjs for anything substantial is bound to land somewhere that mentions peggyjs at some point.

**Comment by jclark-dot-org** [comment]
@lionel-rowe  Thanks for posting this!  I was shocked to find pegjs.org hosting a garbage "domain auction" site; I might not have found peggy.js without this issue.  I'm swapping my dependencies now.

**Comment by Mingun** [comment]
Maybe it is possible to inform npm that `pegjs` dependency is abandoned and users should switch to `peggyjs`?

**Comment by bd82** [comment]
It has been > 9 years since last release of `pegjs`.

I removed pegjs from my the [JavaScript Parsing Libraries Benchmark](https://chevrotain.io/performance/)
and explicitly listed `peggy` as as fork of `pegjs`

<img width="1542" height="1036" alt="Image" src="https://github.com/user-attachments/assets/4f3491c5-5ea5-4459-82cb-71c2031ca087" />
